src/Styles/Home.scss

```diff
@@ -0,0 +1,8 @@
+@import '_variables.scss';
```

Comment:
Getting sassy really quick with the variables. Really love how efficient this is.

Reply:
This is definitely a smart thing to keep things dynamic and easier down the road.